pants

```diff
@@ -126,8 +125,9 @@ function determine_pants_version {
 
   pants_version="$(get_pants_config_value 'pants_version')"
   if [[ -z "${pants_version}" ]]; then
-    pants_version="$(curl -sSL https://pypi.python.org/pypi/pantsbuild.pants/json |
-      "${python}" -c "import json, sys; print(json.load(sys.stdin)['info']['version'])")"
+    die "Please explicitly specify the \`pants_version\` in your \`pants.toml\` under the
```

Comment:
Could we warn with the version that we used instead?

Reply:
That warning would happen every single run, which is super noisy. I don't think that would be that much better of a UX. What's your thinking for not wanting to fail fast?

Reply:
How does this interact with PANTS_SHA?

Reply:
Right above, we have:

```
  if [ -n "${PANTS_SHA:-}" ]; then
    # get_version_for_sha will echo the version, thus "returning" it from this function.
    get_version_for_sha "$PANTS_SHA"
    return
  fi
```

So it's good to go.